Tests/Cell.py

```diff
@@ -1599,7 +1667,7 @@ def test_execute_view_csv(self):
 
         # check if coordinates are the same
         self.assertTrue(coordinates.issubset(self.target_coordinates))
-        values = [float(record.split(',')[3]) for record in records if record != '']
+        values = [float(record.split(',')[4]) for record in records if record != '']
```

Comment:
I'm seeing **index** getting returned part of the `execute_mdx_csv` method...

Reply:
I'm sorry I don't understand 🤔

Reply:
I think there is an issue here. The original test was alright. 

I just checked out this branch and debugged the test. Now the produced CSV doesn't map to the view data anymore.

![image](https://user-images.githubusercontent.com/15182131/98733058-44be6500-23a0-11eb-8e31-fd0ec3b6ba69.png)